notebook/manager.py

```diff
@@ -0,0 +1,103 @@
+"""Manager to read and modify config data in JSON files.
```

Comment:
Maybe call this `config_manager.py`?

Reply:
yeah, wasn't sure where to put it also, but wanted to start with a clean dup of the original PR. Shall I keep it in the root dir?

Reply:
I think so